parsing/dialects/dialect1/parserImpls.ftl

```diff
@@ -2007,18 +2007,18 @@ int VariableBinaryTypePrecision() :
 void LikeAnyAllSome(List<Object> list, Span s) :
 {
     final SqlOperator op;
-    boolean isLike = true;
+    boolean notLike = false;
     SqlNodeList nodeList;
 }
 {
-    [ <NOT> { isLike = false; } ]
+    [ <NOT> { notLike = true; } ]
     <LIKE>
     (
-        <SOME> { op = SqlStdOperatorTable.like(SqlKind.SOME, isLike); }
+        <SOME> { op = SqlStdOperatorTable.like(SqlKind.SOME, notLike); }
```

Comment:
nit: It would probably clean things up just to have a single call to SqlStdOperatorTable.like() below all of these cases. And to just set the SqlKind value within the cases.